net-misc/gns3-server/gns3-server-2.2.8.ebuild

```diff
@@ -0,0 +1,63 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+PYTHON_COMPAT=( python3_{6,7,8} )
+
+inherit distutils-r1
+
+DESCRIPTION="GNS3 server to asynchronously manage emulators"
+HOMEPAGE="http://www.gns3.net/"
+SRC_URI="https://github.com/GNS3/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="qemu"
+
+RDEPEND="
+	~dev-python/aiohttp-3.6.2[${PYTHON_USEDEP}]
+	~dev-python/aiohttp-cors-0.7.0[${PYTHON_USEDEP}]
+	~dev-python/aiofiles-0.4.0[${PYTHON_USEDEP}]
+	>=dev-python/async_generator-1.1.0[${PYTHON_USEDEP}]
+	~dev-python/async_timeout-3.0.1[${PYTHON_USEDEP}]
+	sys-apps/busybox
+	>=dev-python/distro-1.3.0[${PYTHON_USEDEP}]
+	>=app-emulation/dynamips-0.2.18
+	>=dev-python/jinja-2.7.3[${PYTHON_USEDEP}]
+	python_targets_python3_7? ( ~dev-python/jsonschema-2.6.0[${PYTHON_USEDEP}] )
+	python_targets_python3_8? ( ~dev-python/jsonschema-3.2.0[${PYTHON_USEDEP}] )
+	~dev-python/psutil-5.6.6[${PYTHON_USEDEP}]
+	~dev-python/py-cpuinfo-5.0.0[${PYTHON_USEDEP}]
+	>=dev-python/raven-5.23.0[${PYTHON_USEDEP}]
+	>=net-misc/ubridge-0.9.14
+	qemu? ( app-emulation/qemu )
```

Comment:
This doesn't seem to have any function in this ebuild.
https://projects.gentoo.org/qa/policy-guide/dependencies.html#pg0001

Reply:
a GNS3 user could easily argue that gns3-server is basically useless without some virtualization KVM, qemu being the most commonly used

Reply:
I'll drop the dep though for simplicitys sake

Reply:
You should use `optfeature` for it, as stated in the link. This kind of usage is banned by QA.

Reply:
thank you, I was unaware of this in eutils